ubuntu_dockerfile

```diff
@@ -154,7 +154,7 @@
 
 ##########################################################################
 
-FROM ubuntu:latest
+FROM ubuntu:18.04
```

Comment:
Can you please add a comment above to explain why we need to have this version? Also, add a TODO comment to change it to `latest` once the backed is running on python 3.

Reply:
Should I assign the TODO to myself?

The presubmits require format `TODO(username): XXX`

Reply:
You can create an issue on github and follow this template:
`TODO(#<issue-id>): <description>`